graph/registry.go

```diff
@@ -59,7 +60,9 @@ func NewV2Repository(repoInfo *registry.RepositoryInfo, endpoint registry.APIEnd
 		Transport: authTransport,
 		Timeout:   15 * time.Second,
 	}
-	endpointStr := endpoint.URL + "/v2/"
+
```

Comment:
Just a "nit", but for a cleaner change, it would be good to not add the extra whitespace (blank lines)